README.md

```diff
@@ -6,7 +6,8 @@
 - Different types of merging code in GitHub namely:
   - Merge Commit
   - Rebase and Merge
-  - Squash and Merge
+  - Squash and Merge  
+  <a href="https://drive.google.com/uc?export=view&id=1TqQEmimySSZtcFO2tWBBEdgGmlDUizZr"><img src="https://drive.google.com/uc?export=view&id=1TqQEmimySSZtcFO2tWBBEdgGmlDUizZr" style="width: 650px; max-width: 100%; height: auto" title="Click to enlarge picture" />  
```

Comment:
Let's use imgur.com for image hosting or have the image here.

Reply:
okay i will see imgur.com

Reply:
Waiting! 😁

Reply:
@mriganksingh1 I'll update it.